core-blocks/paragraph/index.js

```diff
@@ -287,7 +279,23 @@ const supports = {
 	className: false,
 };
 
+const deprecatedSchema = {
```

Comment:
I'm not sure I understand why these were extracted, from my point of view the `custom*` attributes should remain in the schema because they are still being used. The `width` attribute should not though and should only be added to the deprecated version's attributes.